### DISCUSSION TITLE
Mitteilungen nicht in eine Grafik einbetten.

### DISCUSSION THREAD

**Discussion by michie164** [community/community#65623] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Bitte keine Bilder von Texten für wichtige Informationen. Texte in Bildern muss man zuerst lokal Speichern wenn man den Text elektronisch Übersetzen will. Dann OCR und dann kann man es erst lesen. Warum glaubt ihr das JEDER Mensch alles was in Englisch geschrieben ist versteht? 
NEIN, ich brauche einen Translator!
Also BIIIIIITE Text oder gleich in mehreren Sprachen.

**Comment by Akash1134** [comment]
Hi @michie164 , thanks for participating.

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
